package.json

```diff
@@ -952,8 +952,8 @@
           "mod-settings.entries.collection.get",
           "mod-settings.entries.item.post",
           "mod-settings.entries.item.put",
-          "mod-settings.global.read.ui-inventory.display-settings",
-          "mod-settings.global.write.ui-inventory.display-settings"
+          "mod-settings.global.ui-inventory.display-settings.view",
+          "mod-settings.global.ui-inventory.display-settings.create"
```

Comment:
@Dmytro-Melnyshyn need your confirmation - is it ok to rename these perms in such a way?

Reply:
This might work, but it needs to be tested and the naming doesn't match the `mod-settings` [documentation](https://github.com/folio-org/mod-settings?tab=readme-ov-file#mod-settings).